### ISSUE TITLE
Can I change the confidence level of anchors while training

### ISSUE THREAD

**Issue by ahsan856jalal** [AlexeyAB/darknet#1078]
@AlexeyAB  
I want to know that what is the default confidence level / threshold for anchor to be selected if it detects a object in the frame ?
I want to increase the threshold near to 1 so that my training could be a bit tight. As I am working on fish dataset and fish classes have quite similar features and even corals at the back resembles some of the fish as the dataset is very blur and occluded. So I am thinking that high confidence level of anchor selection might help

**Comment by AlexeyAB** [comment]
What do you mean by `threshold for anchor`? Is it about `-thresh 0.25`?


By default is used `-thresh 0.25`: https://github.com/AlexeyAB/darknet/blob/455b2fc06ff590914f3a1fb13261510d2b95de2e/src/detector.c#L1212

**Comment by ahsan856jalal** [comment]
No, I mean when we are training, the anchors will be computer against the
input data. There will a threshold or confidence, above which the algorithm
considers the raw pixels as foreground and background for vice versa.
Do you mean there is a variable 'threshold'  in train_detector definition
in detector.c which is doing the similar thing during training? So can I do
like :
. /darknet detector train obj.data fish.cfg fish.weights - thresh 0.4 (for
40 percent threshold)
